CHANGELOG.md

```diff
@@ -1,5 +1,90 @@
 # Changelog
 
+## [1.21.0] - UNRELEASED
+
+In release 1.20.x profiles could incorrectly ascribe host-independent components
+to the host architecture, resulting in surprising behaviour with `rust-src`.
+We have [corrected this][pr#2087] and [added mitigations][pr#2115] which should
+mean that as of this release, such incorrect ascriptions are supported and also
+automatically corrected on toolchain update.
+
+We have [officially deprecated][pr#2100] the `rustup install` and `rustup uninstall`
+aliases for `rustup toolchain install` and `rustup toolchain uninstall` respectively.
+Currently we plan to remove support (making using these subcommands an error)
+in 1.22.0.
+
+Due to the large number of confusions around the `complete` profile, we have
+[introduced a warning][pr#2138] if you use it. It's really only meant for
+developers _of_ Rust, or those exploring particular issues in `nightly`.
+
+There are also a large number of other changes, the hilights of which are below.
```

Comment:
'hilight' is such a common misspelling of 'highlight' that it's probably on its way to just being a variant. (May even already be so in some Englishes?)

Reply:
Well spotted, I think that was laziness on my part.